### PR TITLE
Submit SignIn on Return and break out Style

### PR DESCRIPTION
**Pull request by eduardoboucas**

### CODE REVIEW

File: dadi/frontend/components/DropdownItem/DropdownItem.jsx [hunk 1]
```diff
@@ -2,17 +2,11 @@
...
-import { Style } from 'lib/util'
+import Style from 'lib/Style'
 import styles from './DropdownItem.css'
 
 export default class DropdownItem extends Component {
   handleClick(event) {
-
-    // Don't stop propagation for anchor node
-    if (!this.props.href) {
-      event.stopPropagation()
```
Comment (mingard) [inline dadi/frontend/components/DropdownItem/DropdownItem.jsx]: @eduardoboucas what have you done instead to stop this breaking Router?

Comment (eduardoboucas) [inline dadi/frontend/components/DropdownItem/DropdownItem.jsx]: Erm, I didn't mean to remove this. Sorry, I'll fix tomorrow.

Comment (mingard) [inline dadi/frontend/components/DropdownItem/DropdownItem.jsx]: re-ping - this part of the next batch of updates from you?

Comment (eduardoboucas) [inline dadi/frontend/components/DropdownItem/DropdownItem.jsx]: Yes, but you can still merge this PR. It was the presence of `stopPropagation()` (which I removed) that was breaking the router. In fact, I'm not sure we need it anymore, so all should be good.

Comment (mingard) [inline dadi/frontend/components/DropdownItem/DropdownItem.jsx]: Ah, excellent. Then no need to add this back then? Merged

Comment (eduardoboucas) [inline dadi/frontend/components/DropdownItem/DropdownItem.jsx]: Not for now. I'll dig deeper once I revisit the `ButtonWithOptions` component.

**Pull request merged**